config/config.go

```diff
@@ -19,6 +19,7 @@ import (
 )
 
 const DefaultTerragruntConfigPath = "terragrunt.hcl"
+const DefaultTerragruntJsonConfigPath = "terragrunt.hcl.json"
```

Comment:
There's probably a few places where you need to update the usage of `DefaultTerragruntConfigPath` to also look for `DefaultTerragruntJsonConfigPath`. At a minimum, `FindConfigFilesInPath` (
https://github.com/gruntwork-io/terragrunt/blob/master/config/config.go#L192) needs to be updated to look for the json flavor so that `xxx-all` works. This adds some complexity to the logic, but if we are to support `json`, we need to make sure all the commands work or else I can see a lot of issues.